modules/ROOT/pages/policy-mule4-jwt-validation.adoc

```diff
@@ -155,6 +158,35 @@ Ignore this field if you selected *none* as JWT Signing Method.
 Note that all DataWeave expressions must return a boolean value or they will always fail.
 |===
 
+== Obtaining a policy compatible PEM public key
+The policy uses public rsa key in pem format. In this section we explain how to obtain the public.pem file that contains said key.
+Remember that once the public.pem key is obtained, before pasting it to the policy configuration to remove the *-----BEGIN PUBLIC KEY-----* header and the *-----END PUBLIC KEY-----* footer.
```

Comment:
Before applying the public.pem key to the policy configuration, ensure that you remove the following header and footer:
.  *-----BEGIN PUBLIC KEY-----* header 
. *-----END PUBLIC KEY-----* footer